tests/test_init.py

```diff
@@ -74,6 +76,21 @@ def test_load_w_string_path(self):
         string_path = str(self.simple)
         self.assertEqual(self.simplePVL, pvl.load(string_path))
 
+    def test_loadu(self):
```

Comment:
I think a test for when the user passes in a urllib request in would be useful

Reply:
Yeah, it probably would.  I'm not too worried about that one, since the `url` argument basically gets passed straight from the `loadu` function signature to `urllib.request.urlopen` and we don't do anything with it in between or after.

Reply:
So, if there's something wrong with it, then `urllib.request.urlopen()` is the function that will throw the appropriate exception.  I think we just trust it to do that.